CHANGES.md

```diff
@@ -3,6 +3,7 @@
 New themes:
 
 - *NNFX* by [Jim Mason][]
+- *lioshi* by [lioshi][]
```

Comment:
Sorry to nitpick, but is it `lioshi` or `Lioshi`... the comment differs from the nam in the changelog.

I feel like they should be consistent... otherwise, this is approved.